hkpy/hkbase/observer/notification.py

```diff
@@ -0,0 +1,26 @@
+###
+# Copyright (c) 2019-present, IBM Research
+# Licensed under The MIT License [see LICENSE for details]
+###
+
+from enum import Enum, unique
+
```

Comment:
Add `__all__` to control what is being exposed.

Reply:
done